clients/java/signalr/src/main/java/HubConnection.java

```diff
@@ -5,18 +5,18 @@
 import java.util.HashMap;
 
 public class HubConnection {
-    private String _url;
-    private Transport _transport;
+    private String url;
+    private Transport transport;
```

Comment:
![https://media.giphy.com/media/AhAysobj49aqQ/source.gif](https://media.giphy.com/media/AhAysobj49aqQ/source.gif)

Reply:
nice and tidy